### DISCUSSION TITLE
Back-up to (self-hosted) NAS

### DISCUSSION THREAD

**Discussion by Vashiru** [ente-io/ente#1211] [Enhancements]
I recently discovered Ente and I absolutely love it. But as with all cloud services I'm afraid of it suddenly disappearing. I would be interested in a feature where you can add a NAS to your Ente account as a back-up location.

I know the server recently got open sourced, but setting that up requires quite a bit of effort. My request would be a simpler version where you just run it on your NAS (or in a Docker container wherever), point it to a folder where it can write to and just have an authenticated endpoint you can add to your Ente account.

This might be tricky with the E2E encryption, but I still wanted to throw the idea out there.

Perhaps the alternative could be adding your own additional S3 endpoint to your account.

**Comment by mnvr** [comment]
This is already possible! You can use [Ente's CLI](https://github.com/ente-io/ente/tree/main/cli#start-export) to export your data in a cron job to a location of your choice. The exports are incremental, and will also gracefully handle interruptions.

Alternatively, you can use the [Ente Desktop app](https://ente.io/download/desktop) to set up automatic and continuous exports which'll keep running in the background. For more details, see [help.ente.io/migration/export](https://help.ente.io/photos/migration/export/)

I know it is not exactly the same as what you're describing, but in terms of functionality, that is giving you the same result. Your data lives in a place fully under your control. Many of our customers use it - they have a regular Ente subscription for their day to day use, + additional exports to deal with scenarios where they're "afraid of it suddenly disappearing".

**Reply by Vashiru** [reply]
Man, I didn't know I could love Ente even more! In terms of result that's indeed what I'm looking for. I'll have a go at it. Thanks for pointing me the right way.

**Reply by PackElend** [reply]
> This is already possible! You can use [Ente's CLI](https://github.com/ente-io/ente/tree/main/cli#start-export) to export your data in a cron job to a location of your choice. The exports are incremental, and will also gracefully handle interruptions.
> 
> Alternatively, you can use the [Ente Desktop app](https://ente.io/download/desktop) to set up automatic and continuous exports which'll keep running in the background. For more details, see [help.ente.io/migration/export](https://help.ente.io/photos/migration/export/)
> 
Could link or describe it in https://ente.io/faq/features/backup/ as well as Google search gets you there.
